framework/src/main/java/org/checkerframework/framework/util/typeinference8/types/VariableBounds.java

```diff
@@ -287,6 +287,28 @@ public void addConstraintsFromComplementaryBounds(BoundKind kind, AbstractType s
         }
       }
     }
+    if (s.isUseOfVariable()) {
+      UseOfVariable otherVar = (UseOfVariable) s;
```

Comment:
Eiher `s` or `otherVar` is an unexpected name.  Can you choose names that are more similar to one another and convey the meaning?